packages/icon/package.json

```diff
@@ -0,0 +1,22 @@
+{
+	"name": "@ag.ds-next/icon",
+	"version": "1.0.1",
+	"main": "dist/ag.ds-next-icon.cjs.js",
+	"module": "dist/ag.ds-next-icon.esm.js",
+	"license": "MIT",
+	"dependencies": {
+		"focus-trap-react": "^8.9.0"
+	},
+	"peerDependencies": {
+		"@ag.ds-next/box": "1.0.1",
```

Comment:
The icon component itself doesn't appear to need `box`. Keep this as a `devDependency` only.

```suggestion
```